Configuration/AlCa/python/autoCond.py

```diff
@@ -2,60 +2,60 @@
 
     ### NEW KEYS ###
     # GlobalTag for MC production with perfectly aligned and calibrated detector for Run1
-    'run1_design'       :   '91X_mcRun1_design_v2',
+    'run1_design'       :   '92X_mcRun1_design_v0',
     # GlobalTag for MC production (pp collisions) with realistic alignment and calibrations for Run1
-    'run1_mc'           :   '91X_mcRun1_realistic_v2',
+    'run1_mc'           :   '92X_mcRun1_realistic_v0',
```

Comment:
hello @slava77 

does it make sense to deploy these PF corr in the run1 MC workflows ?

We pondered this with the JetMEt contact and concluded not to deploy for heavy ion, pA and run1. In the 2 former cases the reco sequence is qualitatively different from where this update is intended to be consumed, hence a clear choice.

In the case of run1, both the detector (phase1 vs phase0) and the com were  different, hence, imho, also no need to update.

Please advise.